.github/dependabot.yml

```diff
@@ -6,6 +6,9 @@ updates:
     directory: "/"
     schedule:
       interval: "weekly"
+    ignore:
+    # Must match transitive version from org.jenkins-ci.plugins:subversion:
+    - dependency-name: org.tmatesoft.svnkit:svnkit-cli
```

Comment:
then why don't we just depend on the version transitively?  are their conflicts - or is there a `depenencyManagement` that is lost due to [MNG-5761](https://issues.apache.org/jira/browse/MNG-5761)?

Reply:
> then why don't we just depend on the version transitively?

`subversion` depends only on `svnkit` (an old version) and does not pass along the `svnkit-cli` dep.

> are [there] conflicts

Yes, which is why #239 failed.

Reply:
We could also pick up https://github.com/jenkinsci/subversion-plugin/pull/276 via https://github.com/jenkinsci/bom/pull/1672 but that would still be a bit too old. Would need to go up to https://github.com/jenkinsci/subversion-plugin/pull/278 via https://github.com/jenkinsci/bom/pull/1789. :eyes: